parser/lexer_test.go

```diff
@@ -121,7 +121,7 @@ func (s *testLexerSuite) TestComment(c *C) {
 
 	table := []testCaseItem{
 		{"-- select --\n1", intLit},
-		{"/*!40101 SET character_set_client = utf8 */;", int(';')},
+		{"/*!40101 SET character_set_client = utf8 */;", set},
```

Comment:
previously, comment is ignored in parser, so the first token is ';' symbol
now comment start with /*! is parsed as mysql special code, so we get 'set' keyword here